assets/app/view/game/map.rb

```diff
@@ -17,7 +17,9 @@ class Map < Snabberb::Component
       needs :opacity, default: nil
       needs :show_coords, default: Lib::Storage['show_coords'], store: true
       needs :show_location_names, default: (s = Lib::Storage['show_location_names']).nil? ? true : s, store: true
+      needs :map_zoom, default: (s = Lib::Storage['map_zoom']).nil? ? 1 : s, store: true
```

Comment:
refactor this it's bad, i didn't catch it in the previous review but the defaults should not be accessed like this

you have have them in the render method and set default values

Reply:
https://github.com/tobymao/18xx/pull/2440

Reply:
https://github.com/tobymao/18xx/pull/2440 fixes the defaults for the properties that I added; when it's merged, @daniel-sousa-me, you could update this PR to match the pattern that's used there. I suggest waiting till it's merged in case it changes between now and then :wink:

Reply:
Yeah, I'll wait. I just realise that rebasing now to your branch would probably fail, because merging a PR changes the commits.